.circleci/config.yml

```
@@ -155,6 +155,47 @@ jobs:
           name: Publish package
           command: npm publish ./lib/
 
+  deployment:
+    docker:
+      - image: cimg/node:16.15
+        auth:
+          username: $DOCKER_USER
+          password: $DOCKER_PASS
+    working_directory: ~/fun
+    steps:
+      - add_ssh_keys:
+          fingerprints:
+            - "cd:48:76:dd:42:15:64:18:7c:1a:31:2b:d6:ab:60:57"
+      - *checkout_fun
+      - *restore_node_modules
+      - run:
+          name: build Playground for production
+          command: |
+            VERNA_EXAMPLE=verna-examples-playground yarn build:example
+      - run:
+          name: Create the deployment directory copy the git repo there & checkout the deployment branch
+          command: |
+            mkdir ~/deployment
+            cp -R .git ~/deployment/
+            cd ~/deployment
+            git fetch
+            git checkout pages-deployment
+      - run:
+          name: Clean the deployment directory & copy the build results
+          command: |
+            rm -rf ~/deployment/*
+            mkdir ~/deployment/basic
+            cp -R examples/playground/dist/* ~/deployment/basic/
```

To ease things, I think its better to use the same directory name as the example
```suggestion
            cp -R examples/playground/dist/* ~/deployment/playground/
```